README.md

```diff
@@ -49,14 +49,24 @@ And to declare custom paste command use:
 let g:system_copy#paste_command='xclip -sel clipboard -o'
 ```
 
+To make the original yank vim command to works with this plugin, one can add the following to
+it's `.vimrc`:
+
+```
+" Mapping y to use copy
+nmap y <Plug>SystemCopy
+" Mapping p to use paste
+nmap p <Plug>SystemPaste
+```
```

Comment:
This is equivalent to setting your Vim clipboard to use the system clipboard. This is actually the core thing I wanted to avoid as I prefer to keep Vim & system clipboard separate. This plugin exists to provide an additional operator for explicitly interacting with the system clipboard. Can you remove this section?

Reply:
Oh, right, that's a good point, I only wanted to add this info on the README as one may want to  customize the keys to copy and paste through `vim-system-copy` and they don't know how to. #14 talks about it. I could change the `y` or `p` keys to the same example on this issue (`Y` and `P`), what do you think?